testing/tutorials/tutorial_3_uncertainty_estimation.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "ivadomed_tutorial-3_uncertainty-estimation.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TOFLk8iPrzkf"
+      },
+      "source": [
+        "### Ivadomed Tutorial 2: **Uncertainty Estimation**\n",
+        "\n",
```

Comment:
so the link says "Tutorial 3 Uncertainty Estimation" but the title of the tutorial says "Tutorial 2". I have fixed this issue and here is the updated link: https://colab.research.google.com/drive/1L8MSyzqZbJbV336KhT3_B9RQEPnLNI2J#scrollTo=uzOt6ugooys4
(I thought it would get updated automatically, but it didn't, hence posting the link here)